website/plugins/customWebpackConfig/index.js

```diff
@@ -24,6 +23,11 @@ module.exports = function customWebpackConfigPlugin(context, options) {
           }),
           new NodePolyfillPlugin({}),
         ],
+        module: {
+          rules: [
+            { test: /\.py$/, loader: 'raw-loader' }
+          ]
+        }
```

Comment:
Is this an artifact of https://github.com/dbt-labs/docs.getdbt.com/pull/2269?

Reply:
Yeah it's leftover from the original PR. I left it as it seems to be required for working with any python files on the site, so could help in the rare chance someone else adds a python file to the site. 

I figured it wouldn't hurt to keep, but if there's any concerns with leaving it i'm definitely open to removing it.